couchbase-operator/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: couchbase-operator
 description: A Helm chart to deploy the Couchbase Autonomous Operator for easily deploying, managing, and maintaining Couchbase Clusters. Couchbase Server is a NoSQL document database with a distributed architecture for performance, scalability, and availability. It enables developers to build applications easier and faster by leveraging the power of SQL with the flexibility of JSON.
-version: 2.1.006
+version: 2.1.007
```

Comment:
Sorry, can you update this 1 more time, there were some chart changes since last time around.

Reply:
@tahmmee I just bumped this up but this should use valid sem ver for the chart version (2.1.8). Per semver, there should be no leading zeros. I had logged a bug around this b/c helm 3.5.2 started to validate this and causes errors with the versions you are using. I was told these chart versions would need to get fixed...

see response from Helm team - https://github.com/helm/helm/issues/9356

```sh
index.go:339: skipping loading invalid entry for chart "couchbase-operator" "2.0.2-01" from C:\Users\lapellan\AppData\Local\Temp\helm\repository\couchbase-index.yaml: validation: chart.metadata.version "2.0.2-01" is invalid
index.go:339: skipping loading invalid entry for chart "couchbase-operator" "2.0.1-001" from C:\Users\lapellan\AppData\Local\Temp\helm\repository\couchbase-index.yaml: validation: chart.metadata.version "2.0.1-001" is invalid
```